src/region/__tests__/RegionAnnotations-test.tsx

```diff
@@ -107,6 +108,21 @@ describe('RegionAnnotations', () => {
                 expect(defaults.createRegion).toHaveBeenCalledWith({});
             });
         });
+
+        describe('handleAnnotationSelect()', () => {
+            test('should call setActiveAnnotationId if annotation has changed', () => {
+                instance.handleAnnotationSelect('123');
+
+                expect(defaults.setActiveAnnotationId).toHaveBeenCalledWith('123');
+            });
+
+            test('should not call setActiveAnnotationId if annotation has changed', () => {
+                const wrapper2 = getWrapper({ activeAnnotationId: '123', staged: {} as CreatorItem });
```

Comment:
Can probably just update the props on `wrapper.prop('activeAnnotationId', '123')`?